CHANGELOG.txt

```diff
@@ -7,6 +7,11 @@ Changes:
      instrument response (see #2677)
  - obspy.clients.filesystem:
    * add get_waveforms_bulk() method to SDS client (see #2616, #2626)
+ - obspy.io
+   * add support to resove the SEED id of picks for nlloc hyp files and
```

Comment:
```suggestion
   * add support to resolve the SEED id of picks for nlloc hyp files and
```

Reply:
This was already fixed. Did you review the latest version?